README.md

```diff
@@ -1,9 +1,14 @@
 # zppy
 
-**zppy** (pronounced zippee) is a post-processing toolchain for E3SM written 
-in Python. The goal of zppy is to speed-up the post-processing of E3SM 
+**zppy** (pronounced zippee) is a post-processing toolchain for E3SM written
+in Python. The goal of zppy is to speed-up the post-processing of E3SM
 simulations by automating commonly performed tasks.
 
```

Comment:
If you want, you can include the status of `build_workflow.yml` and `release_workflow.yml`,
```
[![CI/CD Build Workflow](https://github.com/E3SM-Project/zppy/actions/workflows/build_workflow.yml/badge.svg)](https://github.com/E3SM-Project/zppy/actions/workflows/build_workflow.yml)
[![CI/CD Release Workflow](https://github.com/E3SM-Project/zppy/actions/workflows/release_workflow.yml/badge.svg)](https://github.com/E3SM-Project/zppy/actions/workflows/release_workflow.yml)
```